pkg/agent/controller/egress/egress_controller.go

```diff
@@ -490,6 +501,37 @@ func (c *EgressController) unrealizeEgressIP(egressName, egressIP string) error
 	return nil
 }
 
+func (c *EgressController) uninstallEgressQoSMeter(meterID uint32) error {
+	if meterID == 0 {
```

Comment:
When would this happen? Does 0 have some specific meaning?

Reply:
We use the mark assigned to `EgressIP` as `meterID`. 0 means this Egress IP is a remote IP. The rate-limiting only happens on the Egress Node. So while uninstalling an Egress on a non-Egress Node, this condition will be hit. Let me add some comments here.